### PR TITLE
Missing migrations for Infobip, Pinpoint?

### PR DESCRIPTION
**Pull request by kaapstorm**
Are we missing migrations for the new backends? I couldn't find them ... but they are working? Aren't they?

Hitting you up for a review @esoergel because I can't ask @nemanja1040 who wrote these models. But @nemanja1040 please feel free to review.

### CODE REVIEW

**Review by esoergel — APPROVED** [review]
Huh, yeah, it looks like the PR that added these models doesn't contain a migration.
https://github.com/dimagi/commcare-hq/pull/27709/files

**Comment by kaapstorm** [comment]
(Turns out the models work fine because their data is persisted using their superclass `SQLMobileBackend`. Subclasses of `SQLMobileBackend` don't add any of their own fields.)

**Review by millerdev — APPROVED** [review]

File: corehq/apps/case_importer/tests/test_importer.py [hunk 1]
```diff
@@ -207,11 +321,39 @@ def testCaseLookupDomainCheck(self):
...
+            ['importer-test-external-id', 'age-1', 'sex-1', 'location-1'],
+            ['importer-test-external-id', 'age-2', 'sex-2', 'location-2'],
+        )
+
+        # the first one should create the case, and the remaining two should update it
+        res = do_import(file, config, self.domain)
+        self.assertEqual(1, res['created_count'])
+        self.assertEqual(2, res['match_count'])
+        self.assertFalse(res['errors'])
+        self.assertEqual(2, res['num_chunks']) # the lookup causes an extra chunk
```
[the remark this answers is not in the document]
Comment (kaapstorm) [reply corehq/apps/case_importer/tests/test_importer.py:348]: Ignoring

**Review by esoergel — APPROVED** [review]
(didn't read over the test changes closely)

**Comment by kaapstorm** [comment]
(Apologies for rewriting history. I wanted to drop the irrelevant test changes and the merges from master that were all futile attempts to get the build to pass.)

**Review by esoergel — COMMENTED** [review]
:hiking_boot: :construction:

**Comment by kaapstorm** [comment]
Travis is consistently failing with this:
```
======================================================================
ERROR: test suite for <class 'corehq.apps.accounting.tests.test_autopay.TestBillingAutoPay'>
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/vendor/lib/python3.6/site-packages/nose/suite.py", line 210, in run
    self.setUp()
  File "/vendor/lib/python3.6/site-packages/nose/suite.py", line 293, in setUp
    self.setupContext(ancestor)
  File "/vendor/lib/python3.6/site-packages/nose/suite.py", line 316, in setupContext
    try_run(context, names)
  File "/vendor/lib/python3.6/site-packages/nose/util.py", line 471, in try_run
    return func()
  File "/mnt/commcare-hq-ro/corehq/apps/accounting/tests/test_autopay.py", line 34, in setUpClass
    super(TestBillingAutoPay, cls).setUpClass()
  File "/mnt/commcare-hq-ro/corehq/apps/accounting/tests/test_invoicing.py", line 72, in setUpClass
    date_end=subscription_end_date,
  File "/mnt/commcare-hq-ro/corehq/util/test_utils.py", line 39, in inner
    return fn(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/accounting/tests/generator.py", line 108, in generate_domain_subscription
    subscription.save()
  File "/mnt/commcare-hq-ro/corehq/apps/accounting/models.py", line 1162, in save
    publish_domain_saved(domain)
  File "/mnt/commcare-hq-ro/corehq/apps/domain/shortcuts.py", line 37, in publish_domain_saved
    producer.send_change(topics.DOMAIN, _domain_to_change_meta(domain_obj))
  File "/mnt/commcare-hq-ro/corehq/apps/change_feed/producer.py", line 61, in send_change
    raise KafkaPublishingError(e)
corehq.form_processor.exceptions.KafkaPublishingError: KafkaTimeoutError: Failed to update metadata after 60.0 secs.
```

But it runs fine locally. And I can't see what it has to do with this change. Other than Nick P and Emord, only @orangejenny and @proteusvacuum have touched the test suite. Hello! I know your changes were either several years ago, or unrelated. But you don't have any ideas for me, do you?

**Pull request merged**